internal/exec/runner.go

```diff
@@ -179,14 +181,45 @@ func (r *runner) Files(args []string) error {
 	return nil
 }
 
-func (r *runner) Compile(args []string, dryRun bool) error {
+func (r *runner) Compile(args []string, dryRun bool, outputPath string) error {
 	meta, err := r.getMeta(args, 1)
 	if err != nil {
 		return err
 	}
 	r.printAffectedFiles(meta)
-	_, err = r.compile(false, false, dryRun, meta)
-	return err
+
+	if outputPath == "" {
```

Comment:
would be really nice if `prototool compile --out -` was understood to mean stdout, so you could do e.g. `prototool compile --out - | yab -transport grpc -descriptor-set - ...`